### DISCUSSION TITLE
Dubai Poolside Networking: Darwinia's Side Event at Token 2049 - Hosted at Furqan's Villa 🏠

### DISCUSSION THREAD

**Discussion by nblogist** [ringdao-community-guild/discussion#122] [Proposals Temperature Check]
⠀⠀⠀⠀⣴⡄⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⣿⡇⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⣿⣿⣦⣄⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⣿⣿⠈⠻⣷⡀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⢀⣀⣿⣿⣶⣶⣾⣿⣶⠶⠀⠀⠀⠀⠀⠀
⠀⠀⠈⠛⣿⣿⡟⠛⠛⠛⢻⣦⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⣿⣿⣿⣿⠏⢀⣾⣿⣆⠀⠀⠀⠀⠀
⠀⠀⠀⠀⣿⣿⡿⠃⣰⣿⣿⣿⣿⡄⠀⠀⠀⠀
⠀⠀⠀⠀⣿⣿⣷⣀⣀⣀⣀⠀⢸⣷⠀⠀⠀⠀
⠀⠀⠀⠀⣿⡟⣿⣿⣿⠟⢁⣴⣿⣿⠀⠀⠀⠀
⠀⠀⠀⠀⣿⡇⣿⡿⠃⣠⣿⣿⣿⣿⡆⠀⠀⠀
⠀⠀⠀⠀⣿⡇⣿⣇⣀⣉⣉⡉⠉⣿⡇⠀⠀⠀
⠀⠀⠀⠀⣿⡇⣿⣿⣿⣿⠋⢀⣴⣿⠁⠀⠀⠀
⠀⠀⠀⠀⣿⡇⢸⣿⠟⢁⣴⣿⣿⣿⠀⠀⠀⠀
⠰⠶⠶⠶⠿⠷⠾⠿⠶⠿⠿⠿⠿⠿⠶⠶⠶⠆

Technically not mine, but it is a villa managed by my very good friend, in the heart of Dubai, and I have previously hosted events with limited number of people, it has a nice view and beautiful pool.

I am keeping this discussion casual to see how the community feels.

After a long and excruciating bear market lasting almost 2 years, we are finally back, BTC broke its last ATH and we are looking towards a very hopeful future. - Now imagine if you can feel it, so can everyone else in the industry. I attended the Permissionless 2023 last year and I remember how low the morale and sentiment overall was there, with a fraction of the people participating despite it being in the hip center of Austin, TX, I recall how eerie it was to walk in the the giant convention center with not much people being present.
Well, all of that is about to change with Token 2049 - Dubai! It is the first major conference that has also gone very well previously in Singapore, it being in Dubai this time allows more attendance esp because it is in _THE_ center of the world accessible by people from the US, Europe, Africa, Europe, East and South Asia! 

This brings me to the preface of a possible proposal: Host a side event in Dubai at the Villa with a Pool! This is a great opportunity to get Darwinia out in front of a global audience, it can get us exposure, more partnerships, integrations, wider access to talent, to name a few upsides! it is a prime time as well as I explained above, and pool villa parties generally attract a lot of people anyways there are several side events taking place on token 2049 [here](https://week.token2049.com/) and none of them will even come close to a mention of "Poolside" on a side event - we can discuss on when to keep - preferably after the first day.

Here are some images of the villa:
[Image 1](https://firealpaca.com/images/get/AlpacaGET_GzgYf5L4.jpg)
[Image 2](https://firealpaca.com/images/get/AlpacaGET_6cVnYWSt.jpg)
[Image 3](https://firealpaca.com/images/get/AlpacaGET_XGFV0d1e.jpg)
[Image 4](https://firealpaca.com/images/get/AlpacaGET_Yozrd5NR.jpg)

CC: @hackfisher @wuminzhe @1022ranji

**Comment by wuminzhe** [comment]
It would be nice to be there.

**Comment by hackfisher** [comment]
Hi @nblogist, it is only 10 days away. I think the preparation is too rushed. I believe that if an offline event hopes to receive support from a funding proposal, it should be prepared at least 1-2 months in advance, and the reasons for needing funding support, such as for purchasing and printing brand promotional materials, covering flights and accommodation, and what will be delivered? For example, a sharing session, should be listed in the temperature check. These key details are missing in the temperature check, could you provide more details if you take it as a temperature check instead of idea? I don't recommend considering this last-minute arrangement. However, if adequately prepared, the next event in Dubai is definitely worth supporting.

**Reply by nblogist** [reply]
I agree @hackfisher this is def rushed - I can still make it but a lot of things will be very dynamic and not preset. This might not be the way for proposal in DCDAO. Perhaps we can think about Token2049 in Singapore or future Dubai events! I appreciate your input on this!
